packages/web3-shared/evm/hooks/useFungibleTokenDetailed.ts

```diff
@@ -11,7 +12,8 @@ export function useFungibleTokenDetailed<
     address: string,
     token?: Partial<P>,
 ): AsyncStateRetry<P | undefined> {
-    const r1 = useNativeTokenDetailed()
+    const _r1 = useNativeTokenDetailed()
+    const r1 = useAsyncRetry(async () => _r1, [address, type, token])
```

Comment:
```suggestion
    const r1 = useNativeTokenDetailed()
    useEffect(r1.retry, [address, type, token])
```



But why `[address, type, token]` would affect to native token?